core/src/main/java/org/apache/carbondata/core/index/IndexInputFormat.java

```diff
@@ -102,6 +103,10 @@
 
   private Set<String> missingSISegments;
 
+  private CdcVO cdcVO;
+
+  private Boolean isCDCJob;
```

Comment:
no need of this boolean, can simply check if CdcVO is null or not and write/read based on that

Reply:
done